src/content/docs/iast/overview.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Introduction to New Relic IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Exploitable vulnerabilities
+metaDescription: Learn what New Relic IAST is and how you can use it.
+redirects:
+    - /docs/iast/use-iast
+---
+
+import iastSummary from 'images/iast_screenshot-crop_summary.webp'
+
+
+<Callout title="PREVIEW"> 
+    This feature is currently in preview. To celebrate it, we're offering you a 3-month free trial! The IAST free trial starts the moment you accept the pre-release software terms. 
+
+    IAST must only be used in a pre-production environment, ideally in a dedicated security environment.
+    
+    Use of the IAST agent may generate additional billable APM data ingest.
+</Callout>
+
+Nowadays, the security of your software applications is essential to protect sensitive data, maintain the trust of your users, and comply with regulatory requirements. New Relic Interactive Application Security Testing (IAST) can help you to prevent cyberattacks and breaches on your applications.
+
+IAST enables you to focus on real security risks, remediate issues faster across the application lifecycle, and ship secure code with speed and confidence. With New Relic IAST, you can see all protected and unprotected applications, which eliminates blind spots and hidden threats. Comprehensive dashboards provide complete visibility into high-priority vulnerabilities, from applications to their components and APIs. 
+
+New Relic IAST is fully integrated with [New Relic Vulnerability Management](/docs/vulnerability-management/overview/), allowing you to continuously find, fix, and verify high-risk vulnerabilities across the software development lifecycle.
+
+New Relic IAST helps you:
+
+- Have a real-time visibility and context into security findings and a guide remediation.
+- Ship code faster with unmatched detection accuracy of security risks.
+- See and secure every application you build and run with full visibility into your code, web components, and configuration data.
+- Provide real-time vulnerability detection and response for any application reducing false positives.
+- Reduce the time and cost to eliminate vulnerabilities earlier in the software development lifecycle.
+- Run both static application security testing (SAST) and dynamic application security testing (DAST) analysis.
+
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > IAST** to open the New Relic IAST page.
```

Comment:
To do what? To sign up for IAST? I think we should make that clear, if so.

And, if that is true, I think it's worth moving this info the the callout at the top. Tell me people that it's a preview they can sign up for and then how to do it.

Reply:
Yes, you're right. Thanks!